### PR TITLE
remove Tiller

### PR DESCRIPTION
**Pull request by rosemarymarano**
And amend cross-refs to https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-operators/#install-helm-and-tiller

### CODE REVIEW

File: docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md [hunk 1]
```diff
@@ -489,15 +489,6 @@ Error: UPGRADE FAILED: Service "external-weblogic-operator-svc" is invalid: spec
 #### Installing an operator and assigning it a service account that doesn't exist
 
 The `helm install` eventually times out and creates a failed release.
-```
-$ helm install kubernetes/charts/weblogic-operator --name op2 --namespace myuser-op2-ns --values o24.yaml --wait --no-hooks
```
Comment (doxiao) [inline docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md:493]: We probably still need this example command line. The statement before this line is based on the fact that there is  —wait in the command line. Without —wait, there is no time-out.

Comment (rosemarymarano) [reply docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md:493]: ok. Are suggesting that I put back just the command:
$ helm install kubernetes/charts/weblogic-operator --name op2 --namespace myuser-op2-ns --values o24.yaml --wait --no-hooks
But, **not** the result:
Error: release op2 failed: timed out waiting for the condition
kubectl logs -n kube-system tiller-deploy-f9b8476d-mht6v
...
[kube] 2018/12/06 21:16:54 Deployment is not ready: myuser-op2-ns/weblogic-operator
...
OR, put back both, the command and the result?

Comment (rjeberhard) [reply docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md:493]: Yes, let's put the command back, but remove the specific output.

**Review by rjeberhard — APPROVED** [review]
Approving assuming we'll come to a happy agreement on the issue we're discussing.

File: docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md (hunk 1 above)
Comment (rosemarymarano) [reply docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md:493]: Done.

**Review by doxiao — APPROVED** [review]

**Pull request merged**